docs/ref/forms/api.txt

```diff
@@ -1043,6 +1043,13 @@ Customizing the error list format
         Defaults to ``None`` which means to use the default renderer
         specified by the :setting:`FORM_RENDERER` setting.
 
+    .. attribute:: field_id
```

Comment:
Do you think it's worth mentioning that this defaults to the auto_id for field errors and no id is associated to non-field errors?

Reply:
I made some changes here, could you have another look? I'm not sure if we need to go further and mention non-field errors.

Reply:
This looks great thank you!